modules/nodes-containers-sysctls-unsafe.adoc

```diff
@@ -102,6 +102,7 @@ spec:
   - name: podexample
     image: centos
     command: ["bin/bash", "-c", "sleep INF"]
+    # TODO: How to update this example, if at all?
```

Comment:
@s-urbaniak @stlaz This already has securityContext set - not sure if the existing settings are important to the section shown (unsafe sysctls)

Reply:
leave as is